cypress/e2e/customTypes/01-duplicate.cy.js

```diff
@@ -10,7 +10,6 @@ describe("Duplicate custom types", () => {
 
   it("when using a label that is already in use it should warn the user", () => {
     cy.visit("/");
-    cy.waitUntil(() => cy.get("[data-cy=create-ct]")).then(() => true);
```

Comment:
Do you know why these were there originally? Any chance it will make the tests flaky?

Reply:
They were here originally for the spinner to finish loading.
I removed them because I didn't see any issues or flakyness while developing my scenario.